### PR TITLE
Add grids for GLLAT and GLMSL

### PR DESCRIPTION
**Pull request by kbevers**
These grids were included in the latest EPSG update

### CODE REVIEW

File: dk_sdfi/dk_sdfi_README.txt [hunk 1]
```diff
@@ -149,3 +149,18 @@ Transform local mean sea level (EPSG:10565) to GVR2016 (EPSG:8267).
 * dk_sdfi_gllmsl_2022.tif
 
 
+### GR96 -> GLLAT
+
+*Source*: [Agency for Data Supply and Infrastructure](https://sdfi.dki/vores-opgaver/geodaesi/koordinattransformation)  
+*Format*: GTG  
+*License*: [Creative Commons Attribution 4.0](https://creativecommons.org/licenses/by/4.0/)
+
+Transform GR966 ellipsoidal heights (EPSG:4909) to GR96+GLLAT (EPSG:10652)
```
Comment (rouault) [inline dk_sdfi/dk_sdfi_README.txt:158]:
```suggestion
Transform GR96 ellipsoidal heights (EPSG:4909) to GR96+GLLAT (EPSG:10652)
```

File: dk_sdfi/dk_sdfi_README.txt [hunk 2]
```diff
@@ -149,3 +149,18 @@ Transform local mean sea level (EPSG:10565) to GVR2016 (EPSG:8267).
...
+
+### GR96 -> GLMSL
+
+*Source*: [Agency for Data Supply and Infrastructure](https://sdfi.dki/vores-opgaver/geodaesi/koordinattransformation)  
+*Format*: GTG  
+*License*: [Creative Commons Attribution 4.0](https://creativecommons.org/licenses/by/4.0/)
+
+Transform GR966 ellipsoidal heights (EPSG:4909) to GR96+GLMSL (EPSG:10651)
```
Comment (rouault) [inline dk_sdfi/dk_sdfi_README.txt:166]:
```suggestion
Transform GR96 ellipsoidal heights (EPSG:4909) to GR96+GLMSL (EPSG:10651)
```

**Comment by kbevers** [comment]
I guess the GitHub action checks have not been updated with the latest EPSG dataset. My local checks of the grids with `check_gtiff_grid.py` using a freshly build `proj.db` doesn't result in any problems.

**Comment by rouault** [comment]
> I guess the GitHub action checks have not been updated with the latest EPSG dataset.

yes the ghcr.io/osgeo/gdal:alpine-normal-latest lags behind. I'm regenerating it, but it will take a few hours

**Comment by kbevers** [comment]
Thanks. I'll try to run the tests again tomorrow

**Comment by rouault** [comment]
Docker image has been regenerated

**Pull request merged**